tasks/prime.py

```diff
@@ -82,3 +82,14 @@ def create_move_task_order(self):
             logger.exception("Non-JSON response")
         else:
             logger.info(f"ℹ️ MoveTaskOrder {json_body['id']} created!")
+
+            move_task_order_id = json_body["id"]
+            e_tag = json_body["eTag"]
+            headers["if-match"] = e_tag
+            resp = self.client.patch(
+                support_path(f"/move-task-orders/{move_task_order_id}/available-to-prime"),
+                headers=headers,
+                **self.user.cert_kwargs,
```

Comment:
This is great and wonderful! However we should add an `name` to this request (and all requests with IDs in the path) so that all of the request to this endpoint are grouped together. Ex: 

```
resp = self.client.patch(
                support_path(f"/move-task-orders/{move_task_order_id}/available-to-prime"),
                headers=headers,
                **self.user.cert_kwargs,
                name=support_path("/move-task-orders/:moveTaskOrderID/available-to-prime"),
            )
```

Reply:
thank you!!